src/components/common/Approve.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <div class="supply-success">
```

Comment:
```suggestion
  <div class="approve">
```
Try using the same name as the file but with dashes